entropy.py

```diff
@@ -4,10 +4,10 @@
 import re
 import string
 
-# For english
+# For English
 ALPHA_SIZE = 26
 
-if len(sys.argv) is not 2:
+if len(sys.argv) != 2:
```

Comment:
"is not" here could technically be false, depending on the Python implementation.